mkdocs.yml

```diff
@@ -14,6 +14,7 @@ nav:
     - Datasets:
       - Datasets: dataset/index.md
       - Quickstart: dataset/quickstart.md
+      - dataset/object-detection.md
```

Comment:
We need a logo as well as a better place for this doc. I think this doc should go under Advanced Usage section and when we have more than one task docs we can create a folder for all the task docs. (Also, remember to merge the latest trunk as the docs hierarchy seems to have changed a bit)

<img width="246" alt="Screenshot 2024-05-27 at 1 57 04 PM" src="https://github.com/kolenaIO/kolena/assets/17771952/87f755a8-0583-4b44-af4b-d7a24aad6896">